Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -500,6 +501,10 @@ render the attributes of a tag. But it's also useful standalone to render extens
 :[key]: (string) A single attribute, array values are joined with whitespace. Boolean values will be rendered as an empty or absent attribute.
 :@allowEmpty: (boolean) Whether empty attributes (HTML5 syntax) should be used for empty, false or null attribute values
 
+.. note:: The ``Neos.Fusion:Attributes`` object is DEPRECATED in favor of a solution inside Neos.Fusion:Tag which takey attributes
```

Comment:
Nice! Should we add the `@deprecated` annotation to the `AttributesImplementation` as well? (Sometimes I jump to the implementation to remind myself of deprecated ones)

Reply:
Not sure, we have to live with this for quite a while.

Reply:
We can live with it being deprecated as well – and since we document that it is deprecated here... :)

Reply:
Added the @deprecated to the AttributesImplementation since we did the same to Array and rawArray etc.

Reply:
+1 for the notice. And make "takey" a "takes"…